examples/BART/BART_introduction.ipynb

```diff
@@ -24,7 +24,7 @@
      "name": "stdout",
```

Comment:
<p>you should remove all the <code>pymc.</code> tags. We will use a different functionality to find notebooks that use pymc functions and classes instead of tags, and leave the tags for "conceptual" topics only. Also, when it comes to the tags, I would choose either the acronym BART or the full name and use only one of them. For barts using the full name might be better</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Just to be clear. I have to remove the tags "pymc3.*" but keep others like "BART"?</p>